vcpkg_windows.txt

```diff
@@ -0,0 +1,5 @@
+boost-build:x86-windows
+boost-math:x64-windows
+boost-math:x86-windows
+boost-math:arm-windows
+boost-math:arm64-windows
```

Comment:
Does this file have to be in the root of the repo? Would `azure-devops` be a better location? It's mentioned by `azure-devops/run_build.yml`.